cpp/ps0002/problem 3 (مساعده بسيطه).cpp

```diff
@@ -2,17 +2,17 @@
 using namespace std;
 int main ()
 {
-unsigned Buildingpopulation='100';
-int Medicationalternatives='50';
-unsigned Numberofteammembers='300';
-short priceofonebaked='5';
-char firstletter ='M';
-double numberofbirths='20000';
-float outputof7/numb='3.5';
-double electricitybill='100500';
-unsigned age='7000000000';
-return 0;
+   unsigned short building_population = 100; // 0.25/0.75
+   unsigned short medication_alternatives = 50; // 0.75/0.75
+   unsigned short number_of_team_members = 300; // 0.75/0.75
+   float price_of_one_baked = 5; // 0.5/0.75
+   char first_letter = 'M'; // 0.75/0.75
+   unsigned int number_of_births = 20000; // 0.5/0.75
+   long double div = 3.5; // 0.25/0.75
+   double electricitybill = 100500; // 0.75/0.75
+   unsigned age = 2000000000; // 0.5/0.75
+   // total 5/6.75
```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [ ] using correct literals and notations for assigning initial values to the variables